src/MB_Toolbox.php

```diff
@@ -13,6 +13,7 @@ class MB_Toolbox
 {
 
   const DRUPAL_API = '/api/v1';
+  const NORTHSTAR = '/v1';
```

Comment:
Sorry, you do use it. Missed it.
Still, it'd be better to rename it to `NORTHSTAR_API`.